docs/data/date-pickers/time-picker/time-picker.md

```diff
@@ -0,0 +1,71 @@
+---
+title: React Time Picker component
+components: DesktopTimePicker, MobileTimePicker, StaticTimePicker, TimePicker, ClockPicker
+githubLabel: 'component: TimePicker'
+packageName: '@mui/x-date-pickers'
+materialDesign: https://material.io/components/time-pickers
+---
+
+# Time Picker
+
+<p class="description">Time pickers allow the user to select a single time.</p>
+
+Time pickers allow the user to select a single time (in the hours:minutes format).
+The selected time is indicated by the filled circle at the end of the clock hand.
+
+## Basic usage
+
+The date picker is rendered as a modal dialog on mobile, and a textbox with a popup on desktop.
```

Comment:
```suggestion
The time picker is rendered as a modal dialog on mobile, and a textbox with a popup on desktop.
```

Reply:
This one was there on the core :D 
Fixed